Formula/e/enzyme.rb

```diff
@@ -1,8 +1,8 @@
 class Enzyme < Formula
   desc "High-performance automatic differentiation of LLVM"
   homepage "https://enzyme.mit.edu"
-  url "https://github.com/EnzymeAD/Enzyme/archive/refs/tags/v0.0.119.tar.gz", using: :homebrew_curl
```

Comment:
the homebrew_curl was added 2.5 years ago, https://github.com/Homebrew/homebrew-core/pull/92070